examples/apollo-fullstack/src/index.ts

```diff
@@ -15,12 +15,18 @@ const schema = makeSchema({
   types,
   outputs: {
     schema: path.join(__dirname, "../fullstack-schema.graphql"),
-    typegen: path.join(__dirname, "../src/fullstack-typegen.ts"),
+    typegen: path.join(
+      __dirname.replace(/\/dist$/, "/src"),
```

Comment:
trick to make sure calling `node dist/schema.js` will find the right dest directory